app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -183,9 +191,11 @@ function initializePodcastPlayback() {
     Array.prototype.forEach.call(records, function (record) {
       var episodeSlug = record.getAttribute('data-episode');
       var podcastSlug = record.getAttribute('data-podcast');
-      record.onclick = function () {
+
+      var togglePodcastState = function (e) {
+        e.preventDefault();
+        var audio = getById('audio');
```

Comment:
suggestion (not blocking) - maybe move this back to its old location inside the `if` statement, given the variable is needed elsewhere